curriculum/challenges/english/14-responsive-web-design-22/learn-css-transforms-by-building-a-penguin/619699c10a0f6e11591d73c4.md

```diff
@@ -16,7 +16,6 @@ You should give `.penguin` a `margin` of `--fcc-expected--`, but found `--fcc-ac
 ```js
 assert.equal(new __helpers.CSSHelp(document).getStyle('.penguin')?.marginRight, 'auto');
 assert.equal(new __helpers.CSSHelp(document).getStyle('.penguin')?.marginLeft, 'auto');
-assert.equal(new __helpers.CSSHelp(document).getStyle('.penguin')?.marginBottom, 'auto');
```

Comment:
This happened because you haven't opened a branch, and didn't clear the history.

Read the docs carefully https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally

it is a guide for dealing with this issue and similar issues